### PR TITLE
fix: clarify error messages

### PR DESCRIPTION
**Pull request by awreccan**
This pull request is to [improve error messages](https://github.com/paypal/selenium-drivex/issues/10). Since developers usually refer to WebElements by their nemo-view `locatorIds`, `locatorId` serves as the best identifier of the element that caused the error.

### CODE REVIEW

**Comment by grawk** [comment]
Thanks for the PR. I think it'll be an improvement to get better default messaging. I have some feedback on the messages then I think we can merge this pretty quickly.

**Comment by awreccan** [comment]
@grawk: I'd be glad to incorporate your feedback into the PR. Do let me know!

**Comment by grawk** [comment]
@awreccan I added some specific requests on changing the error messages to be more accurate. If you can address we can merge.

**Comment by awreccan** [comment]
@grawk Can you link me to these requests? I can't find any.

**Comment by grawk** [comment]
@awreccan I think it was user error on my part. The comments were in a "Pending" state. I submitted the code review as "Changes requested". Can you see them now?

**Comment by awreccan** [comment]
@grawk: updated

**Comment by grawk** [comment]
@awreccan sorry I didn't think to ask earlier, but do you find the messages from `selenium-drivex` are insufficient? E.g.

```
Wed, 08 Feb 2017 03:20:22 GMT selenium-drivex waitForElementVisible name(name) {
    return By.css('*[name="' + escapeCss(name) + '"]');
  } { using: 'css selector', value: '#cc' }
Wed, 08 Feb 2017 03:20:22 GMT selenium-drivex waitForElement name(name) {
    return By.css('*[name="' + escapeCss(name) + '"]');
  } { using: 'css selector', value: 'p.result.good' }
Wed, 08 Feb 2017 03:20:22 GMT selenium-drivex waitForElement name(name) {
    return By.css('*[name="' + escapeCss(name) + '"]');
  } { using: 'css selector', value: 'p.result.bad' }
Wed, 08 Feb 2017 03:20:22 GMT selenium-drivex waitForElementVisible name(name) {
    return By.css('*[name="' + escapeCss(name) + '"]');
  } { using: 'css selector', value: '#ban' }
Wed, 08 Feb 2017 03:20:22 GMT selenium-drivex waitForElement name(name) {
    return By.css('*[name="' + escapeCss(name) + '"]');
  } { using: 'css selector', value: 'p.result.good' }
Wed, 08 Feb 2017 03:20:22 GMT selenium-drivex waitForElementVisible name(name) {
    return By.css('*[name="' + escapeCss(name) + '"]');
  } { using: 'css selector', value: '#ban' }
Wed, 08 Feb 2017 03:20:22 GMT selenium-drivex waitForElement name(name) {
    return By.css('*[name="' + escapeCss(name) + '"]');
  } { using: 'css selector', value: 'p.result.good' }
Wed, 08 Feb 2017 03:20:22 GMT selenium-drivex waitForElementVisible name(name) {
    return By.css('*[name="' + escapeCss(name) + '"]');
  } { using: 'css selector', value: '#email' }
```

**Comment by awreccan** [comment]
@grawk Is that logging from selenium-drivex? I haven't seen logging or errors like that while using nemo, but I think that the `locatorId` is necessary to identify the problematic locator, and just the selenium By selector (css, xpath, or whatever) isn't the best identification.

**Comment by grawk** [comment]
You're right! Sorry I wasn't thinking about it clearly last night. One more adjustment and we're good (please see code review comment).

File: lib/locreator.js [hunk 1]
```diff
@@ -70,10 +70,10 @@ Locreator.prototype.addStarMethods = function addStarMethods(locatorId, locatorJ
     return drivex.present(locator(), parentWebElement);
   };
   locatorObject[locatorId + 'Wait'] = function (timeout, msg) {
-    return drivex.waitForElementPromise(locator(), timeout || 5000, msg);
+    return drivex.waitForElementPromise(locator(), timeout || 5000, msg || 'nemo-view: Wait failed for locator [' + locatorId + ']');
```
Comment (grawk) [inline lib/locreator.js]: You can remove "nemo-view: " from the message. The logger will prepend the module name.

File: lib/locreator.js [hunk 2]
```diff
@@ -70,10 +70,10 @@ Locreator.prototype.addStarMethods = function addStarMethods(locatorId, locatorJ
...
   };
   locatorObject[locatorId + 'WaitVisible'] = function (timeout, msg) {
-    return drivex.waitForElementVisiblePromise(locator(), timeout || 5000, msg);
+    return drivex.waitForElementVisiblePromise(locator(), timeout || 5000, msg || 'nemo-view: WaitVisible failed for locator [' + locatorId + ']');
```
Comment (grawk) [inline lib/locreator.js]: You can remove "nemo-view: " from the message. The logger will prepend the module name.

**Comment by awreccan** [comment]
@grawk addressed!

**Comment by grawk** [comment]
Sorry to be nitpicky @awreccan but please will you add an entry to the CHANGELOG with heading "UNRELEASED"? Then will merge. Thanks!

**Comment by awreccan** [comment]
@grawk opened #62

**Comment by grawk** [comment]
👍

**Pull request merged**